app/spiffworkflow/messages/MessageHelpers.js

```
@@ -878,3 +877,31 @@ export function synCorrleationProperties(
     }
   }
 }
+
+export function deleteMessage(definitions, messageId) {
+  const rootElements = definitions.rootElements;
+
+  // Delete correlation retrieval expressions (Which live outside the message)
+  const corProps = findCorrelationPropertiesByMessage
```

ESLint

[error] 'corProps' is assigned a value but never used. Allowed unused vars must match /_/u.

(no-unused-vars)